lib/config/config.go

```diff
@@ -84,6 +88,14 @@ func LoadConfig() *errco.Error {
 
 	errco.Logln(errco.LVL_D, "msh proxy setup: %s:%d --> %s:%d", ListenHost, ListenPort, TargetHost, TargetPort)
 
+	// set server and protocol version from server JAR file if not specified in config/arguments.
+	// required for backward-compatibility and for minecraft versions without a version.info inside the JAR file.
```

Comment:
what is version.info used for?
what are the differences with Version.json?

Reply:
Whoops, sorry. That should've been `version.json` as well

Reply:
why is it required for `backward-compatibility and for minecraft versions without a version.json inside the JAR file` ?